modules/anti_nd/functions/anti_nd/fn_Init.sqf

```diff
@@ -109,7 +102,7 @@
     },{
         LOG_1("Removing anti_nd event handlers for player = %1",ACE_player);
         params ["_firedEH", "_vehPlayerEH", "_actionVehEH"];
-        player removeEventHandler ["FiredMan", _firedEH];
+        //ACE_player removeEventHandler ["FiredMan", _firedEH];
```

Comment:
Why is this commented?

Reply:
Module no longer removes the FiredMan EH, there is no need